src/app/containers/SignupContainer/SignupContainer.js

```diff
@@ -167,7 +173,7 @@ const SignupContainer = ({ history, onLogin }) => {
                             <PaymentStep
                                 model={model}
                                 paymentAmount={selectedPlan.price.total}
-                                onPaymentDone={handlePaymentDone}
+                                onPay={(...rest) => withLoading(handlePayment(...rest))}
```

Comment:
Do you really need `rest` here?
```suggestion
                                onPay={(data) => withLoading(handlePayment(data))}
```

Reply:
handle payment done requires two parameters: (model, paymentParameters) ...rest seems like a good way to tell that this method doesn't care about params.